sms_sponsorship/controllers/main.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    @author: Quentin Gigon <gigon.quentin@gmail.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+from odoo.addons.cms_form.controllers.main import FormControllerMixin
+from odoo.addons.website_portal.controllers.main import website_account
+from odoo.http import route
+
+
+class SmsSponsorshipWebsite(website_account, FormControllerMixin):
+
+    @route('/sms-sponsorship/<model("recurring.contract"):sponsorship>/',
```

Comment:
add `step2` in the url path (to dissociate from step1 which may have a similar url path)